config/dockerfiles/apiserver/Dockerfile

```diff
@@ -17,11 +17,15 @@ COPY pkg pkg/
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o apiserver cmd/apiserver/apiserver.go
 
+# download Swagger UI files
+RUN git clone https://github.com/swagger-api/swagger-ui -b v2.2.10
```

Comment:
How about setting option `depth` to `1` and deleting `.git` folder to make the container thinner?

Reply:
No need to delete any directories. This is a multi-stage Dockerfile. I only copy the `dist` directory.